tests/components/axis/test_config_flow.py

```diff
@@ -73,9 +69,10 @@ async def test_flow_manual_configuration(hass):
     }
 
 
-async def test_manual_configuration_update_configuration(hass, config_entry):
+async def test_manual_configuration_update_configuration(
+    hass, config_entry, setup_config_entry, mock_vapix_requests
```

Comment:
We normally don't need to set up the config entry for config flow tests. Option flow tests are another matter. There we do need to set up the config entry.

Reply:
These tests are quite old and are definitely not following all current best practices :)

Reply:
Fixed in https://github.com/home-assistant/core/pull/86644